lib/imapoptions

```diff
@@ -1115,11 +1115,20 @@ Blank lines and lines beginning with ``#'' are ignored.
    the same has to be done (cyr_dbtool) for each subscription database
    See improved_mboxlist_sort.html.*/
 
-{ "jmap_emailsearch_db_path", NULL, STRING, "3.1.6" }
+{ "jmap_emailsearch_db_path", NULL, STRING, "3.1.6", "3.5.0" }
```

Comment:
Please use "UNRELEASED" rather than "3.5.0".  I'll fix it to whatever is correct at the time when this reaches a real release.

Reply:
Will do. I still hadn't grokked our versioning system.